client/src/components/Stalker.svelte

```diff
@@ -35,7 +36,11 @@
 </script>
 
 <div class="container">
-  <h1 class="mdc-typography--headline4">Select a chat to stalk</h1>
+  {#if !lobby.alivePlayers.includes($user?.uid ?? "")}
```

Comment:
since you are using the isSpectator prop from chatroom to render this stalker component shouldn't you use to render this as well